RecoTracker/MkFitCore/src/MkBuilder.cc

```diff
@@ -634,6 +637,11 @@ namespace mkfit {
                 continue;
               }
             }
+            // Stop candidates with pT<X GeV
```

Comment:
a minor optimization would be to have this check first, before the dphi check above

Reply:
Done.